webpack.config.ts

```diff
@@ -269,8 +269,8 @@ const appConfig: webpack.Configuration = {
         },
       },
       {
-        test: /\.pegjs/,
-        use: {loader: 'pegjs-loader'},
+        test: /\.pegjs$/,
+        use: ['pegjs-loader?cache=false&optimize=speed'],
```

Comment:
These are default value (See https://github.com/eploko/pegjs-loader?tab=readme-ov-file#pegjs-options)

I set them in here so that they can be parsed by loader-utils here: https://github.com/webpack/loader-utils/blob/v1.4.2/lib/parseQuery.js#L12